Client/Frontend/Reader/ReadabilityService.swift

```diff
@@ -6,29 +6,31 @@ import Foundation
 import Shared
 import WebKit
 
-private let log = Logger.browserLogger
-private let ReadabilityServiceSharedInstance = ReadabilityService()
-
-private let ReadabilityTaskDefaultTimeout = 15
-private let ReadabilityServiceDefaultConcurrency = 1
-
 enum ReadabilityOperationResult {
     case success(ReadabilityResult)
     case error(NSError)
     case timeout
 }
 
 class ReadabilityOperation: Operation {
+    let profile: Profile
+    private let log = Logger.browserLogger
+
     var url: URL
     var semaphore: DispatchSemaphore
     var result: ReadabilityOperationResult?
     var tab: Tab!
     var readerModeCache: ReaderModeCache
 
-    init(url: URL, readerModeCache: ReaderModeCache) {
+    init(
+        url: URL,
+        readerModeCache: ReaderModeCache,
+        profile: Profile
+    ) {
```

Comment:
nit: Not required for string less than 90 chars 
```suggestion
    init(url: URL, readerModeCache: ReaderModeCache, profile: Profile) {
```

Reply:
I do this for clarity actually - not the line length.

Reply:
sorry, clarity how?